Change upload url to configurable

wardragon opened the pull request:
Change upload url to configurable into config.js (and personal config). Added support for root url, so if the dynUpload is not ot be / and request get from /, the application send redirect to browser to point to the defined url. 

This can be used to move the application behind a proxy tha can authenticate only the upload url.

psi-4ward commented:
Reasonable. Pls add a good comment to `config.js` explaining `dynUpload`. 
Moreover, I'll find `dynUpload` not the best name. What about `uploadAppPath` ?

wardragon commented:
So i changed the variable from dynUpload to uploadAppPath as suggested (i agree with you, the syggested name is more suitable). Also added comment to config.js.
